components/src/core/components/CardTable/Cell/Dropdown.vue

```diff
@@ -58,7 +60,7 @@ export default defineComponent({
       // stop event bubbling and prevent emiting event until selected
       e.stopImmediatePropagation();
     },
-    onClick(e: MouseEvent, context: string) {
+    onClick(e: UIEvent, context: string) {
```

Comment:
`onClick` callback is now called upon mouse click as well as hitting the 'enter' key. Therefore, the type of the event type is the generic UIEvent which the `MouseEvent` and `KeyboardEvent` inherits from.